LearnDjango/tests/test_create_accounts.py

```diff
@@ -0,0 +1,303 @@
+# Create Accounts Unit Tests by Terence Lee
+import unittest
+
+import TAInformation.Models.base_user
+from TAInformation.Models.admin import UserAdmin
+from TAInformation.models import User
+from TAInformation.Models.instructor import Instructor
+from TAInformation.Models.ta import TA
+from TAInformation.Models.account_type import AccountType
+
+
+# Functions to setup all tests
+def all_tests_setup(test_user, id_number: int, name: str, password: str, email: str, address: str, phone: str):
+    test_user.user_id = id_number
+    test_user.name = name
+    test_user.password = password
+    test_user.email = email
+    test_user.home_address = address
+    test_user.phone_number = phone
```

Comment:
When saving to the database, User.phone (in models) is an Integer field, so you may have to convert from String to Int, but I'm not sure if python has an explicit conversion

Reply:
I missed this.  The database diagram in the general channel defines it as a string.  I'll ask for clarification on the server and then make the changes needed.